packages/react-scripts/utils/createJestConfig.js

```diff
@@ -19,25 +19,28 @@ module.exports = (resolve, rootDir, isEjecting) => {
 
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
-    moduleNameMapper: {
-      '^.+\\.(ico|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': resolve('config/jest/FileStub.js'),
-      '^.+\\.css$': resolve('config/jest/CSSStub.js')
-    },
     setupFiles: [resolve('config/polyfills.js')],
     setupTestFrameworkScriptFile: setupTestsFile,
     testPathIgnorePatterns: ['<rootDir>/(build|docs|node_modules)/'],
     testEnvironment: 'node',
     testURL: 'http://localhost',
+    transform: {
+      '^.+\\.css$': resolve('config/jest/cssTransform.js'),
+      '^.+\\.(?!(js|jsx|css|json)$)[^\\.]+$': resolve('config/jest/fileTransform.js'),
+    },
+    transformIgnorePatterns: [
+      '/node_modules/.+\.(js|jsx|json)$'
```

Comment:
I needed this because otherwise we can't import CSS from `node_modules` (e.g. `bootstrap/dist/bootstrap.css`). Would it make sense to change the default to that?